packages-exp/auth-exp/src/core/strategies/action_code_settings.ts

```diff
@@ -19,6 +19,7 @@ import { ActionCodeSettings } from '@firebase/auth-types-exp';
 
 import { GetOobCodeRequest } from '../../api/authentication/email_and_password';
 
+/** @internal */
 export function setActionCodeSettingsOnRequest(
```

Comment:
Out of scope but we need to do a pass and make all internal, exported functions start with`_` (looking at you `src/api/**`)

Reply:
ack